src/logtab.cpp

```diff
@@ -807,6 +837,30 @@ void LogTab::ExportToNewTab()
     emit exportToTab(idxList);
 }
 
+void LogTab::ExportToNewTab(COL column)
+{
+    auto selectionList = ui->treeView->selectionModel()->selectedRows();
+    QSet<QString> exportedValues;
+    for (const auto& selected : selectionList) {
+       QString value = selected.model()->index(selected.row(), column, selected.parent()).data().toString();
+       exportedValues.insert(value);
+    }
+
+    const int count = m_treeModel->rowCount();
+    const QModelIndex root;
+    QModelIndexList exportedIdxList;
+    for (int i = 0; i < count; ++i)
+    {
+        auto idx=m_treeModel->index(i, column, root);
```

Comment:
nit: spacing in the assignment "idx=m_treeModel" => "idx = m_treeModel"
This also applies to the line below.